docs/styles.css

```diff
@@ -2707,26 +2707,30 @@
   align-items: center;
   height: calc(100% + 1px);
   margin-bottom: -1px;
-  color: var(--psColorsGray03);
+  color: var(--psColorsGray02);
   font-weight: var(--psTypeFontWeightBook);
   background: none;
   border: 0;
   cursor: pointer;
   overflow: hidden;
-  padding-bottom: 4px;
+  padding: 0 0 var(--psLayoutSpacingTiny) 0;
+  transition: color var(--psMotionSpeedNormal);
```

Comment:
I love this specific targeting of transition.  It help later reading and refactoring, answers "what is this transition meant to do".  Also, just easier to control/predict.